integration/templates/nuxt-node/nuxt.config.js

```diff
@@ -0,0 +1,15 @@
+export default defineNuxtConfig({
+  modules: ['@clerk/nuxt'],
+  // The code below is only necessary in our test environment.
+  // Env vars are automatically read by Nuxt. See https://nuxt.com/docs/guide/going-further/runtime-config
+  runtimeConfig: {
+    public: {
+      clerk: {
+        publishableKey: process.env.NUXT_PUBLIC_CLERK_PUBLISHABLE_KEY,
+      },
+    },
+    clerk: {
+      secretKey: process.env.CLERK_SECRET_KEY,
+    },
+  },
```

Comment:
For some reason, E2E tests are not running without manually declaring the vars here. Will investigate more but this only happens in our test environment